src/writer/autogen.py

```
@@ -119,7 +137,47 @@
+def _get_possible_links(artificial_id_to_component: Dict[str, Dict]):
+    possible_links = []
+    for artificial_id, component in artificial_id_to_component.items():
+        template = writer.abstract.templates.get(component["type"])
+        if not template:
+            continue
+
+        out_ids, dynamic_out = _get_out_ids(template)
+
+        if dynamic_out is not None:
+            field = dynamic_out[1]["field"]
+            out_ids += [f"{dynamic_out[0]}_{key}" for key in json.loads(component["content"][field])]
```

Pylint (3.3.7)

[convention] Line too long (101/100)

(C0301)